demo/plt/wiki10_31K.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import os
+import time
+
+# Parameters
+k = 30938
+kary_tree = 16
+l = 1.0
+passes = 3
+b = 30
+output_model = "wiki10_model"
+
+# Dataset filenames
+train_data = "wiki10_train.vw"
+test_data = "wiki10_test.vw"
+
+print("Vowpal Wabbit PLT example on Wiki10-31K dataset with {} labels".format(k))
+
+# Download dataset
```

Comment:
Perhaps a comment here with a link to the original source of data and how it was converted to vw format?
Specific command would be helpful for conversion would be helpful